src/__experimental__/components/search/search.component.js

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useRef, useLayoutEffect } from 'react';
```

Comment:
If this is a new component, why is it in `__experimental__`?

Reply:
It's built using the `__experimental__ Textbox` so I advised @DipperTheDan  to have it in the same directory

Reply:
Ok - we probably need to start thinking about when we drop the deprecated components so we can move these into the main components folder